htdocs/webportal/class/html.formlistwebportal.class.php

```diff
@@ -201,6 +201,9 @@ public function init($elementEn)
 			$arrayfields['remain_to_pay'] = array('type' => 'price', 'label' => 'RemainderToPay', 'checked' => 1, 'enabled' => 1, 'visible' => 1, 'position' => 10000, 'help' => '',);
 		}
 		$arrayfields['download_link'] = array('label' => 'File', 'checked' => 1, 'enabled' => 1, 'visible' => 1, 'position' => 10001, 'help' => '',);
+		if ($elementEn == "propal") {
+			$arrayfields['signature_link'] = array('label' => 'Signature', 'checked' => 1, 'enabled' => 1, 'visible' => 1, 'position' => 10002, 'help' => '',);
```

Comment:
If online signature is not allowed, we should not add this.
So you should add a test on option PROPOSAL_ALLOW_ONLINESIGN is on.